src/heads-up/views/twitter.vue

```diff
@@ -20,7 +20,7 @@
           class="twitter__preview"
           @load="onResize"
         />
-        <p>Preview based on <external-link href="https://mobile.twitter.com/">mobile.twitter.com</external-link>.</p>
+        <p class="twitter__preview-anchor">Preview based on <external-link href="https://mobile.twitter.com/">mobile.twitter.com</external-link>.</p>
```

Comment:
Maybe the enclosing `<template>` should actually be a `<figure>` and this `p` its `figcaption`? Then a `twitter__preview-caption` class would be fitting.

(I'm struggling with a paragraph having anchor in its name)

Reply:
Agreed